src/OrchardCore.Modules/OrchardCore.Search.Lucene/Services/LuceneIndexManager.cs

```diff
@@ -395,7 +395,7 @@ private async Task WriteAsync(string indexName, Action<IndexWriter> action, bool
                         var config = new IndexWriterConfig(LuceneSettings.DefaultVersion, analyzer)
                         {
                             OpenMode = OpenMode.CREATE_OR_APPEND,
-                            WriteLockTimeout = Lock.LOCK_POLL_INTERVAL * 3
+                            WriteLockTimeout = Lucene.Net.Store.Lock.LOCK_POLL_INTERVAL * 3
```

Comment:
I think this is the required change, please revert the other changes

Reply:
Sorry about that, not sure how those other commits got in there. I will fix this up.